app/models/concerns/notifiable.rb

```diff
@@ -8,13 +8,24 @@ module Notifiable
   def create_notifications(action:, recipients:)
     ActiveRecord::Base.transaction do
       recipients.each do |recipient|
-        Notification.create(
+        notification = Notification.create(
           action: action,
           actor: user,
           notifiable: self,
           recipient: recipient
         )
+
+        broadcast_to_user(notification, user)
```

Comment:
I am an ActionCable newbie.
Are we broadcasting to the correct user? Should be `recipient` ?

Reply:
You are correct.